utils/git.js

```diff
@@ -0,0 +1,66 @@
+const cp = require('child_process');
+const fs = require('fs-extra');
```

Comment:
We make some conventions: `built-in` modules ,`third-part` modules and `self-write` modules are all need separate with each other.

eg there:

```javascript
const cp = require('child_process');

const fs = require('fs-extra');
const minimatch = require('minimatch');
const path = require('path');
const signale = require('signale');

const common = require('./common');
```
What do you think ?

Reply:
Actually it's a widely adopted convention in Python community...Maybe Node.js community has not fully embraced it, let's step forward and do it!